helpers/movimientosCrud/servicios.go

```diff
@@ -59,52 +60,67 @@ func GetMovimientoProcesoExterno(query string, fields string, sortby string, ord
 	return
 }
 
-func CrearMovimientosPublicacion(idMovProcExt int) (movimientosDetalleRespuesta []models.MovimientoDetalle, err map[string]interface{}) {
-	defer errorctrl.ErrorControlFunction("CrearMovimientosPublicacion - Unhandled error!", "500")
+func CrearMovimientoProcesoExterno(detalle []byte) (movimientoProcesoExternoRespuesta models_movimientosCrud.MovimientoProcesoExterno, outputError map[string]interface{}) {
+	defer errorctrl.ErrorControlFunction("CrearMovimientoProcesoExterno - Unhandled error!", "500")
+	tipoMovimientoId, err := beego.AppConfig.Int("tipoMovimientoIdAfectacionCuenPre")
+	if err != nil {
+		outputError = errorctrl.Error("CrearMovimientoProcesoExterno - beego.AppConfig.Int(\"tipoMovimientoIdAfectacionCuenPre\")", err, "500")
+		return models_movimientosCrud.MovimientoProcesoExterno{}, outputError
+	}
+
+	tipoMovimiento := models_movimientosCrud.TipoMovimiento{
+		Id: tipoMovimientoId,
+	}
 
+	// var movimientoEstructura models.MovimientoProcesoExterno
+	nuevoMovimiento := models_movimientosCrud.MovimientoProcesoExterno{
+		TipoMovimientoId: &tipoMovimiento,
+		Activo:           true,
+		Detalle:          string(detalle),
+	}
+
+	// logs.Debug("nuevoMovimiento: ", nuevoMovimiento)
 	urlPublicar := beego.AppConfig.String("movimientos_api_crud_url") +
-		"movimiento_detalle/publicarMovimientosDetalle"
+		"movimiento_proceso_externo/"
```

Comment:
Revisar si esto responde igual
```suggestion
		"movimiento_proceso_externo"
```